src/sentry/lang/javascript/processor.py

```diff
@@ -253,7 +254,7 @@ def fetch_release_file(filename, release):
         else:
             # Write the compressed version to cache, but return the deflated version
             cache.set(cache_key, (releasefile.file.headers, z_body, 200), 3600)
-            result = (releasefile.file.headers, body, 200)
+            result = (releasefile.file.headers, body.decode('utf-8'), 200)
```

Comment:
the changes to fetch_release_file are the primary current bug

requests returns unicode, fetch_release_file was returning bytes